server.js

```diff
@@ -282,6 +282,13 @@ module.exports = function(options) {
         auth: "stripe"
       },
       handler: routes['stripe-dispute']
+    }, {
+      method: "POST",
+      path: "/stripe/charge-succeeded",
```

Comment:
The other webhook doesn't follow this convention, so I think we'd want to handle that change in a new issue if we think it's warranted